src/Components/IndexScene/IndexScene.tsx

```diff
@@ -303,9 +305,10 @@ function getStyles(theme: GrafanaTheme2) {
       '&:first-child': {
         // The wrapper of each filter
         '& > div': {
-          // the 'service_name' filter wrapper
+          // wrapper for all indexed labels
           '&:nth-child(2) > div': {
-            gap: 0,
+            // remove the `gap` from the last label
+            'margin-right': theme.spacing(-2),
```

Comment:
```suggestion
            'marginRight': theme.spacing(-2),
```
Kebab-case not supported warning

Reply:
Done in [d46bc6b](https://github.com/grafana/explore-logs/pull/311/commits/d46bc6bb779e6e83eeb606ba9ea87874c02c4478)